### PR TITLE
fix: [BT-1579]: Added .bazelignore file

### PR DESCRIPTION
**Pull request by AnshuH**
## Checklist
- [ ] I've documented the changes in the PR description.
- [ ] I've tested this change either in PR or local environment.
- [ ] If hashcheck failed I followed [the checklist](https://harness.atlassian.net/wiki/spaces/DEL/pages/21016838831/PR+Codebasehash+Check+merge+checklist) and updated this PR description with my answers to make sure I'm not introducing any breaking changes.

## Comment Triggers
<details>
  <summary>Build triggers</summary>
  
- Feature build: `trigger feature-build`
- Immutable delegate `trigger publish-delegate`
</details>

<details>
  <summary>PR Check triggers</summary>
  
  You can run multiple PR check triggers by comma separating them in a single comment. e.g. `trigger ti0, ti1`
  
- Compile: `trigger compile`
- CodeFormat: `trigger codeformat`
- MessageMetadata: `trigger messagecheck`
- Recency: `trigger recency`
- BuildNumberMetadata: `trigger buildnum`
- runDockerizationCheck: `trigger dockerizationcheck`
- runAuthorCheck: `trigger authorcheck`
- Checkstyle: `trigger checkstyle`
- PMD: `trigger pmd`
- Feature Name Check: `trigger featurenamecheck`
- TI-bootstrap: `trigger ti0`
- TI-bootstrap1: `trigger ti1`
- TI-bootstrap2: `trigger ti2`
- TI-bootstrap3: `trigger ti3`
- TI-bootstrap4: `trigger ti4`
- FunctionalTest1: `trigger ft1`
- FunctionalTest2: `trigger ft2`
- CodeBaseHash: `trigger codebasehash`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

## [Contributor license agreement](https://github.com/harness/harness-core/blob/develop/CONTRIBUTOR_LICENSE_AGREEMENT.md)

- - -
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/38946)

### CODE REVIEW

**Comment by AnshuH** [comment]
trigger messagecheck

**Comment by AnshuH** [comment]
trigger ti0

**Comment by AnshuH** [comment]
trigger cu

**Comment by AnshuH** [comment]
trigger cuBuildNum

**Comment by AnshuH** [comment]
trigger abc cu def

**Comment by AnshuH** [comment]
trigger BuildNum

**Review by abhinav-harness — APPROVED** [review]

**Comment by AnshuH** [comment]
trigger messagecheck

**Comment by AnshuH** [comment]
trigger messagecheck

File: .bazelignore [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+README.md
+VERSION
+buf.yaml
+build.properties
+cd-community
+cert.pem
+data
+docker-files
+docs
+harness-core.log
```
Comment (USB-05) [inline .bazelignore:13]: Seems like this is your local file, please remove this

Comment (AnshuH) [reply .bazelignore:13]: All these files and folders are in harness core

File: .bazelignore [hunk 2]
```diff
@@ -0,0 +1,29 @@
+280-batch-processing
```
[the remark this answers is not in the document]
Comment (AnshuH) [reply .bazelignore]: These are already removed

**Review by USB-05 — APPROVED** [review]
:lgtm:

> Reviewed 1 of 2 files at r2, 1 of 1 files at r5, 1 of 1 files at r6, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/38946#-:-NHwkDJq2HcxoLe3q02P:bnfp4nl)* status: 1. unresolved discussions.

**Review by USB-05 — COMMENTED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/38946#-:-NHwkR817c9UoWhgCFj0:bnfp4nl)* status: 1. unresolved discussions.

**Pull request merged**